### ISSUE TITLE
Intermittent Drops in Extension Activation (POTENTIAL FIX PROVIDED) 

### ISSUE THREAD

**Issue by iiimosley** [sebsojeda/vscode-apple-music#5]
Hey there,
Thanks for putting this extension together for the AM Listeners!

I'm experiencing some intermittent drops in the extension activating in VSCode -- _sometimes it's in the status bar; sometimes it's not_.

Potentially, I think this can be fixed by adding an event to the `activationEvents` property in the `package.json`
https://github.com/sebsojeda/vscode-apple-music/blob/3906c4429d46a331638aa6af97936d04b80ad4e2/package.json#L28

Reviewing the existing docs on [Activation Events](https://code.visualstudio.com/api/references/activation-events), I think [`onStartupFinished`](https://code.visualstudio.com/api/references/activation-events#onStartupFinished) would be an appropriate event.

Lemme know if you need any further details or assistance, here!

**Comment by iiimosley** [comment]
For reference - I successfully tested this change locally. Extensions Host showed the extension as activated on startup finish (previously not shown) and the extension loaded into the status bar.
![image](https://github.com/user-attachments/assets/ef610728-a9fb-4252-9e40-69e345547e78)

**Comment by sebsojeda** [comment]
@iiimosley currently, the extension _should_ activate when you run any command such as "Open" or "Play". Can you confirm this is the case? It's done this way to prevent unnecessary polling of Apple Music if not in use. I do see your point though and maybe the default should be to activate the extension with a config option to show/hide on startup.

**Comment by iiimosley** [comment]
Sorry for the delay!  I appreciate your quick response. I retested the Open & Play commands and they work as expected.

> currently, the extension should activate when you run any command such as "Open" or "Play"

Ahhh, this is where it flubs for me. I am controlling Apple Music via Media Key macros on my keyboard -- not through VSCode keyboard shortcuts. Enables me to control AM no matter which app is in focus.

Namely, I am using the extension to simply **_see_** what's playing rather than controlling what's playing.

> maybe the default should be to activate the extension with a config option to show/hide on startup.

I think this is a solid solution! Showing/hiding on startup would be great config to have.

Similarly, I have added the following automatic task to my User `tasks.json` and it shows the player on open.
```
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "Open Apple Music",
      "command": "${command:vscode-apple-music.showPlayer}",
      "runOptions": {
        "runOn": "folderOpen"
      }
    }
  ]
}
```
